src/styles/card.css

```diff
@@ -0,0 +1,69 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding-top: 32px;
+  padding-bottom: 16px;
+}
+
+.card__image {
+  align-self: center;
+  width: 160px;
+  height: 134px;
+}
+
+.card__product-name {
+  padding: 40px 16px 4px;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+}
+
+.card__product-code {
+  align-self: flex-start;
+  padding-left: 16px;
```

Comment:
instead of setting horizontal padding for every element maybe you should set horizontal padding for the card itself?